### PR TITLE
Updated eslint-config

### PR DESCRIPTION
**Pull request by rapala61**
This pull request updates `eslint-config` with latest changes introduced by @sergii-tkachenko.

Test it by Running:
- npm i
- create or update your `.env` file
- `npm run test`
- `npm run lint`

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: '@dosomething/eslint-config/nodejs/6.x',
+  globals: {
+    app: true
+  },
+  parserOptions: {
```
Comment (sergii-tkachenko) [inline .eslintrc.js]: You shouldn't need this one.

Comment (sergii-tkachenko) [inline .eslintrc.js]: See https://github.com/DoSomething/eslint-config/blob/master/nodejs/8.x.js#L20

**Comment by rapala61** [comment]
@sergii-tkachenko updated!

**Comment by rapala61** [comment]
Thanks! @sergii-tkachenko

**Pull request merged**